README.md

```diff
@@ -15,6 +15,7 @@ To get started on development:
 Several npm scripts have been provided for assisting in development. Each script can be run by navigating to the cloned repository directory in a terminal and executing `npm run scriptname` where `scriptname` is the name of the script you would like to run. The most useful scripts are as follows:
 
 * `package` Builds and packages the extension to a zip file ready to be uploaded to Launch.  
+* `upload` Same as `package` but also uploads the extension package to Launch.  
```

Comment:
I was on the fence on whether I wanted `upload` to ALSO run `package` and I decided I did. It's possible that the person uploading the extension has a problem providing their private key or client secret, which might make it annoying to package the extension every time they try to upload, but I think I'd rather have that then forgetting to package the latest extension code before uploading. Maybe?

Reply:
It's not too weird if you think of it like a build system like make or maven.  Too bad you couldn't only run package if something changed since the last package run.